### PR TITLE
Electrum, Sparrow, Samourai, Blue bounties

### PR DESCRIPTION
**Pull request by Sosthene00**
This is a proposal for opening bounties for the send only feature in the following wallets:
* Sparrow
* Electrum
* BlueWallet
* Samourai

Details must be further discussed before merging

### CODE REVIEW

File: content/docs/bounties.md [hunk 1]
```diff
@@ -44,3 +44,86 @@ As of 05/18/2024, the total payout for this bounty is **0.011 BTC**.
...
+| --- | --- |
+| Anon | 1000k sats |
+
+### Some notes
+
+Given the recent events, some people may have concerns about this bounty, but Samourai being code is in the end nothing more than speech, ie. an idea, and what it stands for is still as true and valuable as ever, people can uphold it for a time, other people can and must uphold it if they also believe in it. We are very much aware of previous antagonisms and online flame wars especially on this issue, and this bounty is not a provocation either, we just want the idea of bitcoin privacy to keep spreading and think that this bounty makes our point.
+
+### Relevant links
+
+- [Existing Rust crate for Silent Payments support](https://github.com/cygnet3/rust-silentpayments)
```
Comment (sethforprivacy) [inline content/docs/bounties.md:86]: I see this crate is mentioned under all of them, is it just a placeholder? I know there are better options for BlueWallet at least, and I know that Samourai is not Rust-based.

File: content/docs/bounties.md [hunk 2]
```diff
@@ -44,3 +44,86 @@ As of 05/18/2024, the total payout for this bounty is **0.011 BTC**.
...
+### Contributors
+
+| Person | Amount |
+| --- | --- |
+| Anon | 1000k sats |
+
+### Relevant links
+
+- [BlueWallet Github](https://github.com/BlueWallet/BlueWallet)
+- [Existing Rust crate for Silent Payments support](https://github.com/cygnet3/rust-silentpayments)
```
Comment (sethforprivacy) [inline content/docs/bounties.md:126]: Should link to BlueWallets library repo.

File: content/docs/bounties.md [hunk 3]
```diff
@@ -44,3 +44,86 @@ As of 05/18/2024, the total payout for this bounty is **0.011 BTC**.
...
+
+### Completion requirements
+
+A merged pull request against whatever will be the active Samourai repository, implementing sending to Silent Payment addresses.
+
+### Contributors
+
+| Person | Amount |
+| --- | --- |
+| Anon | 1000k sats |
```
Comment (sethforprivacy) [inline content/docs/bounties.md:78]:
```suggestion
| Anon | 1M sats |
```

File: content/docs/bounties.md [hunk 4]
```diff
@@ -44,3 +44,86 @@ As of 05/18/2024, the total payout for this bounty is **0.011 BTC**.
...
+
+### Completion requirements
+
+A merged pull request against the Sparrow repository, implementing sending to Silent Payment addresses.
+
+### Contributors
+
+| Person | Amount |
+| --- | --- |
+| Anon | 1000k sats |
```
Comment (sethforprivacy) [inline content/docs/bounties.md:58]:
```suggestion
| Anon | 1M sats |
```

File: content/docs/bounties.md [hunk 5]
```diff
@@ -44,3 +44,86 @@ As of 05/18/2024, the total payout for this bounty is **0.011 BTC**.
...
+
+### Completion requirements
+
+A merged pull request against Electrum repository, implementing sending to Silent Payment addresses.
+
+### Contributors
+
+| Person | Amount |
+| --- | --- |
+| Anon | 1000k sats |
```
Comment (sethforprivacy) [inline content/docs/bounties.md:101]:
```suggestion
| Anon | 1M sats |
```

File: content/docs/bounties.md [hunk 6]
```diff
@@ -44,3 +44,86 @@ As of 05/18/2024, the total payout for this bounty is **0.011 BTC**.
...
+
+### Completion requirements
+
+A merged pull request against BlueWallet repository, implementing sending to Silent Payment addresses.
+
+### Contributors
+
+| Person | Amount |
+| --- | --- |
+| Anon | 1000k sats |
```
Comment (sethforprivacy) [inline content/docs/bounties.md:121]:
```suggestion
| Anon | 1M sats |
```

**Review by sethforprivacy — CHANGES_REQUESTED** [review]

File: content/docs/bounties.md (hunk 1 above)
Comment (Sosthene00) [reply content/docs/bounties.md:86]: Yes it was mostly a placeholder, if you have suggestions for more relevant links we must update it. I still think that mentioning the rust crate is useful at least for reference, and projects written in other languages could even use it directly, it all depends on each project specifics of course as it is not necessarily the most practical approach but there are ffi to many languages. Regarding Samourai in particular I already did some research last year and I think it's possible to use the rust crate, even if a bit annoying.

**Comment by sethforprivacy** [comment]
Note that after merging I noticed some issues with Markdown tables and additional links that I fixed in 2c393dd8ab206a1879b23f4fc54ddab9487f7608.

**Pull request merged**